packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -141,6 +151,14 @@ function Search(props: {
   }
 }
 
+function getZeroIndexStateText(contentNoun?: string) {
+  if (contentNoun) {
+    return `You don't have any ${contentNoun} yet`;
+  }
+
+  return "No options yet";
```

Comment:
definitely more of a designer thing, but this wording feels very sterile. it makes me think of

![image](https://github.com/GetJobber/atlantis/assets/11843215/3f3278d6-11e4-4aab-b7b9-84dffcaee227)


if this was the recommended wording we settled on though, feel free to ignore me!

Reply:
Yeah like I mentioned yesterday in office hours, this is the verbiage we got from Design.  I will still point this out as something to check in the official design review to double check, since this suggestion was mentioned in a slack thread